molecule/logger.py

```diff
@@ -19,11 +19,28 @@
 #  DEALINGS IN THE SOFTWARE.
 
 import logging
+import os
 import sys
 
 import colorama
+from ansible.module_utils.parsing.convert_bool import boolean as to_bool
 
-colorama.init(autoreset=True)
+
+def should_do_markup(fh=sys.stdout):
+    py_colors = os.environ.get('PY_COLORS', None)
+    if py_colors is not None:
+        return to_bool(py_colors, strict=False)
+
+    return (callable(getattr(fh, 'isatty', None)) and
+            fh.isatty() and
+            os.environ.get('TERM') != 'dumb')
+
+
+def colorama_init():
```

Comment:
Can we test this method in conjunction with the other one, please?
Because currently it looks like
![](https://thumbs.gfycat.com/BriefUniformHornet-size_restricted.gif)